stl/inc/functional

```diff
@@ -1479,6 +1479,69 @@ namespace placeholders {
     _INLINE_VAR constexpr _Ph<20> _20{};
 } // namespace placeholders
 
+#if _HAS_CXX20
+// FUNCTION TEMPLATE _Call_front_binder
+template <size_t... _Ix, class _Cv_FD, class _Cv_tuple_TiD, class... _Unbound>
+auto _Call_front_binder(index_sequence<_Ix...>, _Cv_FD&& _Obj, _Cv_tuple_TiD&& _Tpl, _Unbound&&... _Unbargs) {

```

Comment:
I am unsure whether this should have conditional `noexcept` to avoid enforcement machinery being emitted by the compiler. Looking at the generated assembly may help decide.